Add Java sample app content to OIE embedded auth with SDK and SIW sample app guide 2 (#2319)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md

>   If you are using the embedded authentication sample apps, add the following redirect URI `http://localhost:8080` (in addition to the default redirect URI `http://localhost:8080/authorization-code/callback` that is already listed).

Reviewer: Please verify the redirect URI. Note that this default might change in future depending on how the backend Orgs are provisioned.

Author: ```suggestion
  If you are using the embedded authentication sample apps, add `http://localhost:8080` as the redirect URI.
```

Reviewer: looks good.

Author: Discussed with Arvin: decided to remove the text in brackets.
